### DISCUSSION TITLE
Idee per la versione 1.5.6

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#594] [Idee e proposte]
Buonasera io ho quì un'pò di idee per la prossima versione:
1) l'orario
2) la compresenza
3) le assenze del personale
4) il modulo di evaquazione
5) la possibilità di aggiungere PON
6) il voto 0
7) la possibilità di mettere ritardi/uscite multiple es. inserisci evento multiplo >> seleziona alunni (preso da argo)
8) per il docente i voti sotto il 6 in rosso
9) il voto + e - (non 6+/6- ma proprio il voto - o + che non farà media) non so se mi intendo
10) la possibilità di mettere il fuori classe con la spunta DDI in caso di emergenza
11) voto impreparato con valore 4/preparato con valore 6
12) la presa visione per le note
13) nuova schermata voti con la media dell' alunno ispirata ad argo (es. data 11/12 alunno a 4) allego immagine per intendermi
14) richiedere o no la giustifiica dopo un ritardo
15) la possibilità del coordinatore di classe di vedere i compiti e le verifiche assegnate
Buonaserata
![es  schermata voti](https://github.com/iisgiua/giuaschool/assets/137722020/48f5caf7-355d-45c3-92b1-e0ef16e2f547)

**Comment by trinko** [comment]
TIeni presente che:
**2. la compresenza**:
è già possibile; basta configure due docenti sulla stessa cattedra e potranno firmare entrambi.

**5. la possibilità di aggiungere PON**: 
riguarda una gestione amministrativa e non viene gestita dal registro

**14. richiedere o no la giustifiica dopo un ritardo**:
Si può già fare, in due modi. Il primo modo inserendo i minuti per il _ritardo breve_, in modo che i ritardi entro quel numero di minuti non richiedano la giustificazione. Il secondo modo, forzando la giustificazione da parte del docente: il docente, cioè, anziché andare ad inserire che l'alunno non deve giustificare quel ritardo, più semplicemente lo giustifica lui (che mi pare anche formalmente corretto), saltando il meccanismo della giustificazione della famiglia.

I punti restanti li segno come richiesta.

**Comment by ghost** [comment]
Si va bene, ma non è comodo. Cioè, fai un esempio: un insegnante deve
inserire una compresenza per potenziamento di italiano, devi aggiungere la
cattedra di italiano all'insegnante che deve fare potenziamento ogni volta
che è in quella classe? Oltre che non è membro del consiglio di classe
oppure è lì solo una volta, non sempre. Le chiedo gentilmente di rivedere
queste impostazioni, anche dal lato amministrativo, in quanto usare il
registro suo è molto efficace e anche pieno di funzionalità, ma allo stesso
tempo ha molte lacune e non è molto usabile. Per esempio, per l'esame di
stato, perché fare su una centinaia di siti, invece che solo su registro?
Prenda ispirazione da Lampschool, per esempio, e aggiunga le funzioni che
sono presenti in Lampschool. Spero di essere stato comprensibile, signore.

Il giorno sab 2 dic 2023 alle ore 09:53 Antonello Dessì <
***@***.***> ha scritto:

**Reply by Davide031** [reply]
Per la compresenza concordo con lei
